jvm-pdf/src/main/groovy/org/asciidoctor/gradle/jvm/pdf/AsciidoctorPdfTask.groovy

```diff
@@ -49,15 +54,32 @@ class AsciidoctorPdfTask extends AbstractAsciidoctorTask {
         copyNoResources()
     }
 
-    /** The directory where custom are fonts to be found.
+    @Deprecated
+    File getFontsDir() {
+        if (this.fontDirs.size() > 1)
+            throw new Exception("There is more than 1 file in the fonts directory")
```

Comment:
Don't throw `Exception` (I'm surprised COdeNarc did not pull you up on this). I don't think there is another exception available in the classpath that is specific enough, so create a new exception class and throw that. See https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/master/base/src/main/groovy/org/asciidoctor/gradle/base/ModuleNotFoundException.groovy for an example on how we define exception classes in this project.